### ISSUE TITLE
Homepage styles - Create and style middle elements section

### ISSUE THREAD

**Issue by fridaland** [fridaland/InTheFight#25]
### Create and style middle elements section in homepage

Add middle section made up of **3 elements**.  This section should exist below the `About Us` title and description. Each element will contain an image on one side and a placeholder title and description on the other.

<img width="1031" alt="Screen Shot 2020-10-23 at 11 11 49 AM" src="https://user-images.githubusercontent.com/40510669/97027598-92be1680-1520-11eb-92d8-1116a7cc1675.png">

The images can be found under `src/images/home-images`.
Use:
- `home-image-a.png`
- `home-image-b.png`
- `home-image-c.png`

Title and description:
```
Header
Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.
```

See attachments below:
<img width="533" alt="Screen Shot 2020-10-23 at 10 46 50 AM" src="https://user-images.githubusercontent.com/40510669/97027269-162b3800-1520-11eb-998a-9b5740e3e619.png">
<img width="253" alt="Screen Shot 2020-10-23 at 11 05 35 AM" src="https://user-images.githubusercontent.com/40510669/97027274-175c6500-1520-11eb-89cb-926338c32c5e.png">

**Comment by wandersonpsantos** [comment]
can i make this issue?

**Comment by fridaland** [comment]
@wandersonpsantos it's yours! Thanks for contributing!

**Comment by wandersonpsantos** [comment]
first time using gatsby and atom pattern, I don't know if I did it right, waiting for feedback, for necessary adjustments

**Comment by fridaland** [comment]
@wandersonpsantos you did a great job! Closing this as it was resolved by #30
